src/lib.rs

```diff
@@ -429,7 +517,47 @@ where
 
         if self.config.verbose_exit {
             if let Some(span) = ctx.span(&id).and_then(|span| span.parent()) {
-                self.write_span_info(&span.id(), &ctx, SpanMode::PostClose);
+                self.write_span_info(&span.id(), bufs, &ctx, SpanMode::PostClose);
+            }
+        }
+    }
+}
+
+fn scope_path<'a, R: LookupSpan<'a>>(span: &SpanRef<'a, R>) -> ScopeFromRoot<'a, R> {
+    span.scope().from_root()
+}
+
+/// Runs `A` and `B` side by side and only yields items present in `B`
+struct DifferenceIter<L, R, F> {
```

Comment:
This is used for only printing the relevant new spans.

Solving the [Lowest Common Ancestor](https://en.wikipedia.org/wiki/Lowest_common_ancestor) problem for the span tree